src/main/java/com/dataloom/datastore/data/controllers/DataController.java

```diff
@@ -210,21 +237,30 @@ private EntitySetData loadNormalEntitySetData(
                     EnumSet.of( Permission.READ ) );
         }
 
+        LinkedHashSet<FullQualifiedName> orderedPropertyFqns = authzHelper.getAllPropertiesOnEntitySet( entitySetId )
```

Comment:
`EntityType.getProperties()` is a `LinkedHashSet` wrapped by `Collection.unmodifiableSet` right now. Not sure if we really want to remove `Collection.unmodifiableSet`, so I will keep it there for now. Since that's just an wrapper, EdmAuthorizationHelper would still return the property Ids in the right order